src/components/HeroBanner/styles.module.scss

```diff
@@ -1,7 +1,29 @@
 
-.image {
-  object-fit: cover;
+.container {
   width: 100%;
-  max-height: 511px;
-  min-height: 250px;
+  border-bottom: 2px solid #eeeeee;
+  padding: 25px 50px 22px;
```

Comment:
Interesting that these aren't multiples of 8

Reply:
Isn't that a google material design standard? Either way we should probably add defined definitions that live in another file so that they can be consistent across the project

Reply:
Not suggesting it be part of this PR, I think it can come later